admin/src/main/resources/io/buoyant/admin/js/delegator.js

```diff
@@ -29,7 +29,7 @@ var Delegator = (function() {
       var path = $('#path-input').val();
       window.location.hash = encodeURIComponent(path);
       var request = $.get(
-        "/delegator.json?" + $.param({ n: path, d: dtabViewer.dtabStr() }),
+        "/delegator.json?" + $.param({ path: path, dtab: dtabViewer.dtabStr() }),
```

Comment:
yup, agreed. fortunately all the code is in one place. ;)

https://github.com/BuoyantIO/linkerd/issues/328